### PR TITLE
Product selector and refactored App

### PR DESCRIPTION
**Pull request by codingavatar**
Product selector will update the currentProductId. App has been converted to use hooks. Please review, thanks!

### CODE REVIEW

File: client/src/components/Product Selection/Product.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, {useState, useEffect} from 'react';
+import { OtherProducts } from '../styles/OtherProducts.styled';
+
+export default function Product ({productInfo, styles, setCurrentId}) {
+  return (
+    <OtherProducts onClick={() => { setCurrentId(productInfo.id); }}>
+      <h4>{productInfo.name}</h4>
+      <img src={styles.results[0].photos[0].url}
+        width="100px" height="100px" alt={productInfo.name === 'Bright Future Sunglasses' ? '😎' : `${productInfo.name} image`}
```
Comment (poioper) [inline client/src/components/Product Selection/Product.jsx:9]: lol, cool

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -18,35 +19,28 @@ const theme = {
   }
 };
 
-class App extends React.Component {
-  constructor(props) {
```
Comment (poioper) [inline client/src/components/App.jsx:22]: Converted to functional component is much better, thank you!

**Review by poioper — APPROVED** [review]
Looks great!

File: client/src/components/Product Selection/Product.jsx (hunk 1 above)
Comment (codingavatar) [reply client/src/components/Product Selection/Product.jsx:9]: 😎

**Pull request merged**